sycl-fusion/passes/target/TargetFusionInfo.cpp

```diff
@@ -408,8 +408,8 @@ class SPIRVTargetFusionInfo : public TargetFusionInfoImpl {
   }
 
   Function *createRemapperFunction(
-      const Remapper &R, BuiltinKind K, StringRef OrigName, Module *M,
-      const jit_compiler::NDRange &SrcNDRange,
+      const Remapper &R, BuiltinKind K, StringRef OrigName [[maybe_unused]],
```

Comment:
```suggestion
      const Remapper &R, BuiltinKind K, [[maybe_unused]] StringRef,
```

Really just a minor, but I think having the attribute before the type and dropping the argument name is more readable.

Reply:
Ah yes, thanks!